packages/loot-core/src/server/main.ts

```diff
@@ -2411,7 +2424,10 @@ export async function initApp(isDev, socketName) {
   // }
 
   const url = await asyncStorage.getItem('server-url');
-  if (url) {
+  if (url === 'https://not-configured/') {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏could you add a "TODO" comment to clean this up after a few months have passed?